packages/shylu/shylu_node/basker/src/shylubasker_solve_rhs.hpp

```diff
@@ -582,22 +604,39 @@ namespace BaskerNS
       #ifdef BASKER_DEBUG_SOLVE_RHS
       printf("Lower Solve blk (%d, 0): size=(%dx%d) srow=%d, scol=%d\n",b,(int)L.nrow,(int)L.ncol, (int)L.srow,(int)L.scol);
       printf("[\n");
-      for (Int i = 0; i < gn; i++) printf( "%d  %e %e\n",i,x(i),y(i));
+      for (Int i = 0; i < gn; i++) printf( "%d %.16e %.16e\n",i,x(i),y(i));
       printf( "];\n");
+      if (b == 3) {
```

Comment:
Note for later cleanup (future PR if preferred) e.g. dropping the check on specific b value